provider/aws/formation/g1/app.json.tmpl

```diff
@@ -171,6 +171,21 @@
                 }
               ]
             }
+          },
+          {
+            "PolicyName": "PassRolePolicy",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Effect": "Allow",
+                  "Action": [
+                    "iam:PassRole"
+                  ],
+                  "Resource": "*"
```

Comment:
Does it has to be `*`?
gen2 app doesn't need `*`, but why gen1 need that?

Reply:
This is open role according to whichever permission user wants to assign, and can range from any service on aws upto any level the user wishes to assign, as such we cannot limit anything here otherwise it will fail for some Role arn. Gen2 assigns Policy and as such does not need us to do it, as the user already creates their policy and then assigns it but in gen1 we are responsible for giving necessary permissions to cloud formation stack to assign the role to ecs task definition.